### DISCUSSION TITLE
Use Operator from Github

### DISCUSSION THREAD

**Discussion by jairogutierrezpayu** [arkmq-org/arkmq-org-broker-operator#444] [General] [answered]
Hi, i can install operator using Github (Without local clone/install). For example use Kustomizations.yaml files point to github directly?

I suspect some like that in https://github.com/artemiscloud/activemq-artemis-operator/tree/main/config directory, this contains some kustomizations.yaml files.

In this moment i can installed operator locally (repo cloned such like documentation).

**Comment by gaohoward** [comment, marked as answer]
You can use the stuff in deply/install directory, it has all the resources you need to install the operator.
take a look at this script
https://github.com/artemiscloud/activemq-artemis-operator/blob/main/deploy/install_opr.sh
and other scripts for installing a cluster-wise operator.
